application/Espo/Resources/i18n/en_US/Global.json

```diff
@@ -305,7 +305,7 @@
         "fieldNotMatchingPattern$uriOptionalProtocol": "{field} must be a valid URL",
         "fieldInvalid": "{field} is invalid",
         "fieldIsRequired": "{field} is required",
-        "fieldPhoneInvalidCharacters": "Only digits, latin letters and characters `-+_@:#().` are allowed",
+        "fieldPhoneInvalidCharacters": "Only digits, latin letters, and characters `-+_@:#().` are allowed",
```

Comment:
> The Oxford Companion to the English Language notes: "Usage varies as to the inclusion of a comma before *and* in the last item. ... This practice is controversial and is known as the serial comma or Oxford comma, because it is part of the house style of [Oxford University Press](https://en.wikipedia.org/wiki/Oxford_University_Press)."[[15]](https://en.wikipedia.org/wiki/Serial_comma#cite_note-15)